third_party/color_emoji/emoji_builder.py

```diff
@@ -353,6 +353,21 @@ def write_bitmapSizeTable (self, glyph_maps):
 		self.pop_stream ()
 
 
+# copied from nototools/font_data
+_UNICODE_CMAPS = {(4, 0, 3), (4, 3, 1), (12, 3, 10)}
+
+def delete_from_cmap(font, chars):
+    """Delete all characters in a list from the cmap tables of a font."""
+    cmap_table = font['cmap']
+    for table in cmap_table.tables:
+        table_tup = (table.format, table.platformID, table.platEncID)
+        if table_tup in _UNICODE_CMAPS:
```

Comment:
Well, alright. Another concern I had was that there might potentially be other unicode cmaps not covered here. But I suppose if it works in nototools it will work here as well. BTW, why not just import this from nototools?